.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,16 +15,35 @@ Refer to CONTRIBUTING.MD for more details.
   https://github.com/vmware/vic/blob/master/.github/CONTRIBUTING.md
 -->
 
-Fixes #
 
-<!--
-To trigger a custom build with this PR, include one of these in the PR's body:
-- To skip running tests (e.g. for a work-in-progress PR), add `[ci skip]` or `[skip ci]`
-to the commit message or the PR title.
-- To run the full test suite, use `[full ci]`.
-- To run _one_ integration test or group, use `[specific ci=$test]`. Examples:
-  - To run the `1-01-Docker-Info` suite: `[specific ci=1-01-Docker-Info]`
-  - To run all suites under the `Group1-Docker-Commands` group: `[specific ci=Group1-Docker-Commands]`
-- To skip running the unit tests, use `[skip unit]`.
-- To fail fast (make normal failures fatal) during the integration testing, use `[fast fail]`.
--->
+<!--  DO NOT EDIT BELOW. The markdown below is used to provide some coarse options for how CI runs against this PR -->
```

Comment:
Could we have a bats test that validates this template? (E.g., it seems useful to verify that the template mentions each known directive and doesn't mention any unknown directives.)